docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
```

Comment:
TypeScript compiles away "let" - since doc.js is JavaScript - we need to do it ourselves.